### PR TITLE
fix(blockchainAPI): adding history endpoint API

### PR DESCRIPTION
**Pull request by geekbrother**
This PR adds the Blockchain-API history endpoint API description to the documentation.

### CODE REVIEW

File: docs/specs/servers/blockchain/blockchain-api.md [hunk 1]
```diff
@@ -312,11 +312,12 @@ The POST request body should be in JSON format with the following structure:
 
 Used to lookup fungible assets balances
 
-`GET /v1/account/{address}/balance`
+`GET /v1/account/{address}/balance?projectID={projectID}`
```
Comment (chris13524) [inline docs/specs/servers/blockchain/blockchain-api.md]:
```suggestion
`GET /v1/account/{address}/balance?projectId={projectId}`
```

File: docs/specs/servers/blockchain/blockchain-api.md [hunk 2]
```diff
@@ -371,6 +372,61 @@ Used to lookup fungible assets balances
...
+#### Request arguments:
+
+```typescript
+{
+    "projectId": string,
+    "currency"?: string,
+    "cursor"?: string,
+    "onramp"?: string
+}
+```
```
Comment (chris13524) [inline docs/specs/servers/blockchain/blockchain-api.md]:
```suggestion
#### Query parameters:

```typescript
{
    projectId: string,
    currency?: string,
    cursor?: string,
    onramp?: string
}
```
```

**Review by nopestack — APPROVED** [review]
looks good, though please incorporate @chris13524's suggestions about formatting before merging

File: docs/specs/servers/blockchain/blockchain-api.md (hunk 1 above)
Comment (geekbrother) [reply docs/specs/servers/blockchain/blockchain-api.md]: Fixed. Thanks!

File: docs/specs/servers/blockchain/blockchain-api.md (hunk 2 above)
Comment (geekbrother) [reply docs/specs/servers/blockchain/blockchain-api.md]: Fixed. Thanks!

**Pull request merged**